scss/button/_theme.scss

```diff
@@ -1,9 +1,8 @@
 // TODO: button variables
-$button-focused-box-shadow: 0 0 0 2px darken($widget-bg, 6%) !default;
-$button-active-box-shadow: inset 0 3px 5px darken($widget-bg, 9%) !default;
+$button-focused-box-shadow: 0 3px 4px 0 rgba(0, 0, 0, .06) !default;
+$button-active-box-shadow: inset 0 2px 2px 0 rgba(0, 0, 0, .06) !default;
 
-$primary-focused-box-shadow: 0 0 0 2px lighten($primary-button-bg, 23%) !default;
-$primary-active-box-shadow: inset 0 3px 5px desaturate($primary-button-bg, 3%) !default;
+$primary-focused-box-shadow: 0 3px 4px 0 rgba(255, 99, 88, .06) !default;
```

Comment:
It is exactly the $accent color. I was unaware that I could use a hex variable inside an RGBA function in SASS. Corrected now.